app/assets/stylesheets/helpers/_text.scss

```diff
@@ -303,10 +303,10 @@ article {
     margin-bottom: 1.5em;
     padding-bottom: 1em;
 
-    p:first-child {
+    .get-started-intro > p:first-child {
       margin-top: 0;
     }
-
+    
```

Comment:
Some rogue whitespace has been introduced here.
Is your editor configured to strip out trailing whitespace and to show invisibles?